.github/CODEOWNERS

```diff
@@ -714,6 +710,11 @@
 /core/templates/services/auth-backend-api.service*.ts @vojtechjelinek
 
 
+# Android team
+/core/android_validation_constants*.py @seanlip @BenHenning
```

Comment:
Any specific reason for moving the "Android team" below? (Nolte: We have added project sections in alphabetically sorted order!

Reply:
Yes -- I wanted these to be near the critical section so that the correct order of precedence happens. Otherwise, the extensions/ rule would be overridden by the more general /extensions/interactions rule.